src/ln/channel.rs

```diff
@@ -284,6 +284,9 @@ pub struct Channel {
 	pending_inbound_htlcs: Vec<InboundHTLCOutput>,
 	pending_outbound_htlcs: Vec<OutboundHTLCOutput>,
 	holding_cell_htlc_updates: Vec<HTLCUpdateAwaitingACK>,
+
+	pending_update_fee: Option<u64>,
```

Comment:
Can you add a comment above these describing at a high level what the conditions are for updating them?